docs/API.md

```diff
@@ -478,7 +478,10 @@ connected. On binding a socket setting this flag is effective only on this
 socket itself. Note though that there are some post-bound options that have
 important meaning when set prior to connecting.
 
-
+Note that usually options can be set either on a group or on a socket, and
+when set on a group, it will be remembered and then derived by every socket
+that will be automatically created and added to the group. This concerns every
+socket option, unless different rule for that case is declared explicitly.
```

Comment:
Note that options are usually set either on a group or on a socket. When set 
on a group, the options will persist. They will be derived by every socket
automatically created and added to the group. This applies to every
socket option, unless a different rule is declared explicitly for a specific case .

Reply:
Not sure, but I used "can" here intentionally. There are some options that make sense (or are allowed to be set) only on group or only on a socket, although most of them can be set on both groups and sockets. Please refer.